src/RelSeries.jl

```diff
@@ -275,11 +275,22 @@ end
 
 @enable_all_show_via_expressify SeriesElem
 
-function show(io::IO, a::SeriesRing)
-   print(io, "Univariate power series ring in ", var(a), " over ")
-   print(IOContext(io, :compact => true), base_ring(a))
+function show(io::IO, ::MIME"text/plain", a::SeriesRing)
+  print(io, "Univariate power series ring in ", var(a), " with precision ", a.prec_max)
+  println(io)
```

Comment:
Genuine question: Why not:
```suggestion
  println(io, "Univariate power series ring in ", var(a), " with precision ", a.prec_max)
```
Not that I mind either way, but I wonder if there is a difference here that I am missing, because elsewhere you also used `print(io,"FOO");print(io,"\nBAR")` and I was wondering why not `println(io,"FOO");print(io,"BAR")`